aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -5,17 +5,19 @@ description: Use .NET Core Generic Host in ASP.NET Core apps.  Generic Host is r
 monikerRange: '>= aspnetcore-3.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 4/17/2020
+ms.date: 11/09/2021
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: fundamentals/host/generic-host
 ---
 # .NET Generic Host in ASP.NET Core
 
 :::moniker range=">= aspnetcore-6.0"
 
-The ASP.NET Core templates create a .NET Core Generic Host (<xref:Microsoft.Extensions.Hosting.HostBuilder>).
+This article provides information on using the .NET Generic Host in ASP.NET Core.
 
-This article provides information on using .NET Generic Host in ASP.NET Core. For information on using .NET Generic Host in console apps, see [.NET Generic Host](/dotnet/core/extensions/generic-host).
+The ASP.NET Core templates create a <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder> and <xref:Microsoft.AspNetCore.Builder.WebApplication>, which configure, build, and wrap a .NET Generic Host. For more information on configuring the .NET Generic Host using `WebApplicationBuilder` and `WebApplication`, see <xref:fundamentals/minimal-apis>.
```

Comment:
I'm not sure the minimal API's link is the best link. How about [New hosting model](https://docs.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-6.0&tabs=visual-studio#new-hosting-model) ?

Reply:
Yes, agreed. I didn't know that was there, but I think it's a much better fit.

Reply:
I think mentioning that `WebApplication` wraps the generic host is mostly an implementation detail that doesn't really matter to the developers using it. Maybe this could be rewritten as follows.

```suggestion
The ASP.NET Core templates create a <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder> and <xref:Microsoft.AspNetCore.Builder.WebApplication>, which proved a streamlined way to configure in run web applications without a `Startup` class. For more information on configuring the .NET Generic Host using `WebApplicationBuilder` and `WebApplication`, see <xref:migration/50-to-60#new-hosting-model>.
```

I'm not sure if the updated link is valid. Eventually, it would probably be best to add a new a section for `WebApplication` under `fundamentals/host/` and link to that.